modular_skyrat/modules/moretraitoritems/code/game/objects/items/drinks/drinkingglass.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/reagent_containers/food/drinks/drinkingglass/shotglass/syndicate
+	name = "shot glass"
+	desc = "A shot glass - the universal symbol for terrible decisions."
+	icon_state = "shotglass"
+	base_icon_state = "shotglass"
+	gulp_size = 50
+	amount_per_transfer_from_this = 50
+	possible_transfer_amounts = list(50)
+	volume = 50
```

Comment:
Not sure how I feel about this, it just feels like something that wouldn't really lead to a lot of interaction with the players, 50u is a really big amount for something so stealthy, although, to be fair, don't they tell how much liquid they contain?

Reply:
They shouldn't say, no. They're mostly a thing I've wanted to bring from goon for a really long time, and could lead to some interesting bartending shinanigans.

Reply:
I just have a feeling that it won't really contribute much to roleplay, someone will eventually put a very deadly mix into those and still have space for some regular booze so people wouldn't expect it and would get like 20u of a very deadly mix of chemicals. I don't know how good that'd be for roleplay, honestly.